trinity/utils/eip1085.py

```diff
@@ -0,0 +1,277 @@
+import json
+from pathlib import Path
+from typing import (
+    Any,
+    cast,
+    Dict,
+    Iterable,
+    NamedTuple,
+    Tuple,
+    Type,
+    Union,
+)
+
+from mypy_extensions import (
+    TypedDict,
+)
+
+from jsonschema import (
+    validate,
+    ValidationError as JSONSchemaValidationError,
+)
+
+from eth_typing import (
+    Address,
+    BlockNumber,
+    Hash32,
+    HexAddress,
+    HexStr,
+)
+
+from eth_utils import (
+    big_endian_to_int,
+    decode_hex,
+    to_canonical_address,
+    to_dict,
+    to_int,
+    to_tuple,
+    ValidationError,
+)
+from eth_utils.toolz import (
+    sliding_window,
+)
+
+from eth.typing import AccountDetails
+from eth.vm.base import BaseVM
+from eth.vm.forks import (
+    TangerineWhistleVM,
+    FrontierVM,
+    HomesteadVM as BaseHomesteadVM,
+    SpuriousDragonVM,
+    ByzantiumVM,
+    ConstantinopleVM,
+)
+
+
+T_RAW_EIP1085 = Dict[str, Any]
```

Comment:
@pipermerriam just hinting that the `T_*` notation is usually associated with type params (used with generics) and not for type aliases which should use `PascalCase` as other regular classes